docs/users/developer-tools.md

```diff
@@ -24,17 +26,19 @@ Please note this will not intercept emails if your application is configured to
 MailHog provides several [configuration options](https://github.com/mailhog/MailHog/blob/master/docs/CONFIG.md). If you need to alter its configuration, you can do so by adding the desired environment variable to the `environment` section for the web container in the `.ddev/docker-compose.yaml` for your project.
```

Comment:
Pretty sure we should lose this scary paragraph.

Reply:
Eek! gone.